src/Command/CreateCommand.php

```diff
@@ -91,6 +93,17 @@ protected function execute(InputInterface $input, OutputInterface $output): int
         $this->migrationService->setIO($io);
         $this->createService->setIO($io);
 
+        /** @var string|null $path */
+        $path = $input->getOption('path');
+
+        /** @var string|null $namespace */
+        $namespace = $input->getOption('namespace');
+
+        if ($path !== null || $namespace !== null) {
```

Comment:
`$namespace` has priority. If both options specified migration path will be used. It is also for configuration.
We can throw the `Exception` in this place 

https://github.com/yiisoft/db-migration/blob/1ec213d71cd3bf88415ce8f70536b03955252a0b/src/Service/MigrationService.php#L61-L67

It will affect both options and configurations.

Reply:
Good idea. Suggest make it in separate PR.